### PR TITLE
add form names if not "normal"

### PR DESCRIPTION
**Pull request by quiddity-wp**
Tested and working in my server

### CODE REVIEW

File: config/defaults/dts.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "template": {
       "embed": {
         "color": "{{ivcolor}}",
-        "title": "{{round iv}}% {{name}} cp:{{cp}} L:{{level}} {{atk}}/{{def}}/{{sta}} {{boostemoji}}",
+        "title": "{{round iv}}% {{name}}{{#if form}}{{#isnt formname 'Normal'}} {{formname}}{{/isnt}}{{/if}}  cp:{{cp}} L:{{level}} {{atk}}/{{def}}/{{sta}} {{boostemoji}}",
```
Comment (123FLO321) [inline config/defaults/dts.json:9]: Is the extra whitespace indented? 
Between the `{{/if}}` and `cp:{{cp}` there are 2 whitespaces. (`{{/if}}  cp:{{cp}}`)

File: config/defaults/dts.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
     "template": {
       "embed": {
         "color": "{{color}}",
-        "title": "?% {{name}} {{boostemoji}}",
+        "title": "?% {{name}}{{#if form}}{{#isnt formname 'Normal'}} {{formname}}{{/isnt}}{{/if}}  {{boostemoji}}",
```
Comment (123FLO321) [inline config/defaults/dts.json:25]: same here

File: config/defaults/dts.json [hunk 3]
```diff
@@ -37,7 +37,7 @@
     "platform": "discord",
     "template": {
       "embed": {
-        "title": "Raid against {{name}} has started at {{gymName}}!",
+        "title": "Raid against {{name}}{{#if form}}{{#isnt formname 'Normal'}} {{formname}}{{/isnt}}{{/if}}  has started at {{gymName}}!",
```
Comment (123FLO321) [inline config/defaults/dts.json:40]: and here

File: config/defaults/dts.json (hunk 1 above)
Comment (quiddity-wp) [reply config/defaults/dts.json]: ty, fixed

**Pull request merged**